doc/contributing/docs/localization/guide.rst

```diff
@@ -0,0 +1,135 @@
+Tarantool Translation Guidelines
+================================
+
+Use this guide when localizing Tarantool into Russian.
+
+Tone of voice
+-------------
+
+We address IT specialists fairly knowledgeable in their respective fields.
+The goal of our translations is to help these people understand how to use Tarantool.
+Think of us as their colleagues and address them as such.
+Be professional but friendly.
+Don’t command or patronize.
+Use colloquial speech but avoid being too familiar.
+It’s all about the golden mean.
+
+Think twice when translating modal verbs.
+Avoid using expressions like «вы должны», because they sound like a demand in Russian,
+and «вам придётся», because it implies that our readers will face a lot of trouble.
+Make it easy for the user to read the documentation.
+
+Use gender-neutral expressions like «сделать самостоятельно» instead of «сделать самому», etc.
+
+
+Term choice
+-----------
+
+Though not all of our readers may be fluent in English,
+they write in English-based programming languages
+and are used to seeing error messages in English.
+Therefore, if they see an unfamiliar and/or more archaic Russian term
+for a familiar concept, they might have trouble correlating them.
+
+We don’t want our audience to feel confused, so we prefer newer terms.
+We also provide the English equivalent for a term
+if it is used in the article for the first time.
+
+If you feel like the older Russian term may sound more familiar for a part of the audience
+(for example, those with a math background),
+consider adding it in parentheses along with the English equivalent.
+Don’t repeat the parentheses throughout the text.
+A similar rule applies to
+:ref:`introducing terms in Tarantool documentation <introduce-terms>`.
+
+Term choice examples
+~~~~~~~~~~~~~~~~~~~~
+
+..  container:: table
+
+    .. list-table::
+       :widths: 24 38 38
+       :header-rows: 1
+
+       *   -
+           -   First time
+           -   All following times
+       *   -   state machine
+           -   машина состояний (конечный автомат, state machine)
+           -   машина состояний
+       *   -   “write-ahead log (WAL)” at first entry, “WAL” later
+           -   журнал упреждающей записи (write-ahead log, WAL)
+           -   журнал упреждающей записи; WAL; журнал WAL *(using a descriptor)*
+       *   -
+           -
+           -
+
+Best practices
```

Comment:
I'd use third-level headings with paragraphs throughout this section, instead of a list. When one's makes a review and wants to point to a particular rule, it's much easier to do with a heading than with a list item.